src/api/app/controllers/comments_controller.rb

```diff
@@ -8,7 +8,9 @@ def index
   end
 
   def create
-    @obj.comments.create!(body: request.raw_post, user: User.session!, parent_id: params[:parent_id])
+    comment = @obj.comments.new(body: request.raw_post, user: User.session!, parent_id: params[:parent_id])
+    authorize comment, :create?
```

Comment:
I believe we should test the policy `:create` method

Reply:
I haven't checked if this policy is tested... just wanted to fix the endpoint. I will come up with a little test for this

Reply:
It is not. Maybe in another PR that this will depend on?

Reply:
I create the specs in this PR @danidoni